config_defaults.yml

```diff
@@ -556,46 +556,43 @@ Rakefile:
 Gemfile:
   required:
     ':development':
-      # hardcode JSON version to what's shipped in pdk for now
-      - gem: json
-        version: '= 2.0.4'
-        condition: "Gem::Requirement.create('~> 2.4.2').satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: json
-        version: '= 2.1.0'
-        condition: "Gem::Requirement.create(['>= 2.5.0', '< 2.7.0']).satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: json
-        version: '= 2.3.0'
-        condition: "Gem::Requirement.create(['>= 2.7.0', '< 2.8.0']).satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: 'puppet-module-posix-default-r#{minor_version}'
-        version: '~> 1.0'
+      - gem: 'json'
+      - gem: 'voxpupuli-puppet-lint-plugins'
+        version: '>= 3.0.0'
+      - gem: 'facterdb'
+        version: '< 2.0.0'
+      - gem: 'metadata-json-lint'
+        version: '< 4.0.0'
+      - gem: 'puppet-lint'
```

Comment:
Isn't this redundant since `voxpupuli-puppet-lint-plugins' already depends on it:
https://github.com/voxpupuli/voxpupuli-puppet-lint-plugins/blob/56aa3a801627210093203ceeb1e309f291763e7b/voxpupuli-puppet-lint-plugins.gemspec#L17

Reply:
Fair point